src/fragments/lib/graphqlapi/js/working-with-files.mdx

```diff
@@ -0,0 +1,910 @@
+The Storage and GraphQL API categories can be used together to associate a file, such as an image or video, with a particular record. For example, you might create a `User` model with a profile picture, or a `Post` model with an associated image. With Amplify's GraphQL API and Storage categories, you can reference the file within the model itself to create an association.
+
+To get started, run the following in an existing Amplify project:
```

Comment:
@renebrandel @david-mcafee  - do you want to use the fact that you can add the "api" category first with the Cognito user pool as an Auth method here to streamline the number of CLI commands?

Reply:
@abdallahshaban557 - in that case, we'd have to add instructions on adding auth. Telling them to run a single command with default configurations is simpler, IMO. @renebrandel?